mimesis/builtins/da.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: utf-8 -*-
+
+"""Specific data provider for Denmark (da)."""
+
+import random
+
+from mimesis.builtins.base import BaseSpecProvider
+from mimesis.typing import Seed
+
+__all__ = ['DenmarkSpecProvider']
+
+
+class DenmarkSpecProvider(BaseSpecProvider):
+    """Class that provides special data for Denmark (da)."""
+
+    def __init__(self, seed: Seed = None):
+        """Initialize attributes."""
+        super().__init__(locale='da', seed=seed)
+
+    class Meta:
+        """The name of the provider."""
+
+        name = 'denmark_provider'
+
+    def cpr(self) -> str:
+        """Generate a random CPR number (Central Person Registry).
+
+        :return: CPR number.
+
+        :Example:
+            0105865167
+        """
+        day = '{:02d}'.format(random.randint(1, 31))
```

Comment:
Oh, sorry. I forgot that we use our own random, use it like this please:

`day = '{:02d}'.format(self.random.randint(1, 31))`

Sorry again.

Reply:
@danielarnason One more change request and that's all. Thanks!